dvc/lock.py

```diff
@@ -73,7 +74,11 @@ def lock(self):
             except flufl.lock.TimeOutError:
                 raise LockError(
                     "cannot perform the cmd since DVC is busy and "
-                    "locked. Please retry the cmd later."
+                    "locked. You can delete .dvc/lock to fix it but first, "
+                    "please make sure there is no other ongoing DVC process "
+                    "by running: {blue}`ps xaf | grep dvc`{nc}.".format(
+                        blue=colorama.Back.BLUE, nc=colorama.Back.RESET
```

Comment:
I like this `colorama.Back` stuff. Should we create a ticket to apply this in all existing messages with sample commands like this one? (usually surrounded with back tick `, but some times quote ')

Reply:
@jorgeorpinel I don't mind. But do you like the colour as well? :)

Reply:
Opened #2784